site/docs/v1/tech/shared/_scim-limits.adoc

```diff
@@ -1,6 +1,6 @@
 The current SCIM implementation does not support bulk operations using SCIM. Bulk operations on users, such as addition and deletion, are available via link:/docs/v1/tech/apis/[FusionAuth APIs], but are not currently supported via SCIM compatible endpoints. If this functionality is required, please file a https://github.com/fusionauth/fusionauth-issues/issues[GitHub issue] explaining your use case.
 
-The list operation is limited to 10,000 users if you are using the Elasticsearch search engine. This https://github.com/FusionAuth/fusionauth-issues/issues/494[GitHub issue] discusses this limitation.
+The list operation is limited to 10,000 users if you are using the Elasticsearch search engine. See link::/docs/v1/tech/reference/limitations#user-searches[Limitations] for more information
```

Comment:
still a potential issue in scim so just changed the reference from the issue to the limitations page